pkg/cloud/baremetal/actuators/machine/actuator.go

```diff
@@ -51,6 +51,9 @@ const (
 	externalRemediationAnnotation   = "host.metal3.io/external-remediation"
 	poweredOffForRemediation        = "remediation.metal3.io/powered-off-for-remediation"
 	requestPowerOffAnnotation       = "reboot.metal3.io/capbm-requested-power-off"
+	nodeLabelsBackupAnnotation      = "remediation.metal3.io/node-labels-backup"
+	nodeAnnotationsBackupAnnotation = "remediation.metal3.io/node-annotations-backup"
+	nodeFinalizer                   = "machine.openshift.io"
```

Comment:
I was not sure if what value should be put here, let me know if this needs to be changed

Reply:
Something unique to this controller. So probably move to the metal3.io namespace